sdk/evaluation/azure-ai-evaluation/README.md

```diff
@@ -20,88 +32,152 @@ For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
 
-    pprint(relevance_score)
-    # {'gpt_relevance': 5.0}
+```
+
+## Evaluate API
+The package provides an `evaluate` API which can be used to run multiple evaluators together to evaluate generative ai application response.
 
-    # Content Safety Evaluator
+### Evaluate existing dataset
 
-    # Initialize Project Scope
-    azure_ai_project = {
-        "subscription_id": <subscription_id>,
-        "resource_group_name": <resource_group_name>,
-        "project_name": <project_name>
+```python
+from azure.ai.evaluation import evaluate
+
+result = evaluate(
+    data="data.jsonl", # provide your data here
+    evaluators={
+        "blocklist": blocklist_evaluator,
+        "relevance": relevance_evaluator
+    },
+    # column mapping
+    evaluator_config={
+        "relevance": {
+            "column_mapping": {
+                "query": "${data.queries}"
+                "ground_truth": "${data.ground_truth}"
+                "response": "${outputs.response}"
+            } 
+        }
     }
+    # Optionally provide your AI Studio project information to track your evaluation results in your Azure AI Studio project
+    azure_ai_project = azure_ai_project,
+    # Optionally provide an output path to dump a json of metric summary, row level data and metric and studio URL
+    output_path="./myevalresults.json"
+)
+```
+For more details refer to [Evaluate on test dataset using evaluate()][evaluate_dataset]
 
-    violence_eval = ViolenceEvaluator(azure_ai_project)
-    violence_score = violence_eval(query="What is the capital of France?", response="Paris.")
-    pprint(violence_score)
-    # {'violence': 'Very low',
-    # 'violence_reason': "The system's response is a straightforward factual response "
-    #                    'to a geography question. There is no violent content or '
-    #                    'language present.',
-    # 'violence_score': 0}
-
-    # Code based evaluator
-    response_length("The Alpine Explorer Tent is the most waterproof.")
-    # {'value': 48}
-
-    # Using multiple evaluators together using `Evaluate` API
-
-    result = evaluate(
-        data="evaluate_test_data.jsonl",
-        evaluators={
-            "response_length": response_length,
-            "violence": violence_eval,
-        },
-    )
+### Evaluate Generative AI Application
+```python
+from askwiki import askwiki
 
-    pprint(result)
+result = evaluate(
+    data="data.jsonl",
+    target=askwiki,
+    evaluators={
+        "relevance": relevance_eval
+    },
+    evaluator_config={
+        "default": {
+            "column_mapping": {
+                "query": "${data.queries}"
+                "context": "${outputs.context}"
+                "response": "${outputs.response}"
+            } 
+        }
+    }
+)
 ```
+Above code snippet refers to askwiki application in this [sample][evaluate_app].
+
+For more details refer to [Evaluate on a target][evaluate_target]
+
 ### Simulator
```

Comment:
For the TSG section below, could we replace with https://aka.ms/azureaieval-tsg?

Reply:
Done